docs/install.md

```diff
@@ -0,0 +1,72 @@
+# Installing BKPR
+
+A Bitnami Kubernetes Production Runtime (BKPR) release consists of a collection of Kubernetes manifests written in Jsonnet plus the accompanying `kubeprod` installer binary. The kubeprod binary deals with all the platform-specific details, evaluates the Jsonnet manifests, and applies them to the existing Kubernetes cluster.
+
+This document walks you through installing the `kubeprod` binary.
+
+## Installing the `kubeprod` installer
+
+The `kubeprod` binary can be installed from pre-built binary releases or be built directly from source.
+
+### Installing from binary releases
+
+BKPR releases are available for 64-bit versions of Linux, macOS and Windows platforms. Download the latest stable version for your platform of choice from the [releases](https://github.com/bitnami/kube-prod-runtime/releases) page.
```

Comment:
Are there any specific or minimum versions of Windows/macOS that are needed/supported? If yes, we should mention those here.

Reply:
There aren't any such requirements. We build 64 bit static binaries so they should just work regardless of the OS versions / flavors.